substrate/runtime/staking/src/lib.rs

```diff
@@ -381,6 +494,114 @@ impl<T: Trait> Module<T> {
 				.collect::<Vec<_>>()
 		);
 	}
+
+	/// Lookup an T::AccountIndex to get an Id, if there's one there.
+	pub fn lookup_index(index: T::AccountIndex) -> Option<T::AccountId> {
+		let usize_index: usize = As::<usize>::as_(index);
+		let mut set = Self::enum_set(T::AccountIndex::sa(usize_index / ENUM_SET_SIZE));
+		let i = usize_index % ENUM_SET_SIZE;
+		if i < set.len() {
+			Some(set.swap_remove(i))
```

Comment:
this whole `if/else` could be `set.get(i)`

Reply:
but it would `clone`; `swap_remove` doesn't.